examples/vertex_ai_orchestration/config.yaml

```diff
@@ -0,0 +1,15 @@
+name: vertex_example_pipeline
```

Comment:
for other examples you mentioned in the README the option to run with `-c config`, why not here?